docs/recipes.md

```diff
@@ -59,22 +59,24 @@ imageLoader.execute(request)
 Or you can create an OkHttp [`Interceptor`](https://square.github.io/okhttp/interceptors/) that sets headers for every request executed by your `ImageLoader`:
 
 ```kotlin
-class ResponseHeaderInterceptor(
+class RequestHeaderInterceptor(
     private val name: String,
     private val value: String
 ) : Interceptor {
 
     override fun intercept(chain: Interceptor.Chain): Response {
-        val response = chain.proceed(chain.request())
-        return response.newBuilder().header(name, value).build()
+        val request = chain.request().newBuilder()
+            .header(name, value)
+            .build()
+        return chain.proceed(request)
     }
 }
 
 val imageLoader = ImageLoader.Builder(context)
     .okHttpClient {
         OkHttpClient.Builder()
             // This header will be added to every image request.
-            .addNetworkInterceptor(ResponseHeaderInterceptor("Cache-Control", "max-age=31536000,public"))
```

Comment:
Same here re: leaving this as `max-age=31536000` (we can probably remove the `public`).